### DISCUSSION TITLE
Automation - Error when creating New Row [Solved]

### DISCUSSION THREAD

**Discussion by Doctor-N0** [Budibase/budibase#12116] [Help] [answered]
There appears to be sometimes a bug that new rows cannot be created if no value is specified in the Automation--> Create Row fields. The following error appears: 
`"response": "{\"validation\":{\"VariableX\":[\"must be of type number\",\"is not a number\"]}}"`
My workaround was to simply enter a value of 0 in the Automation-->Create Row field.
I am using MySQL, so maybe that's causing this strange issue. It took me quite a while to figure this out, so I thought I'd share it.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Doctor-N0,

I'm going to close this as you have solved it!

I've ran into this myself, the error checking on the number type fields may be slightly too harsh. I'll see If i can bring this up to product to see if anything can be done in this case.

Thanks in advance.
